diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2abf6166..e9e94ddf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -238,8 +238,8 @@ endif()
 
 # Add minimal googletest targets. The provided one has many side-effects, and
 # googletest has a very straightforward build.
-add_library(gtest third_party/googletest/src/gtest-all.cc)
-target_include_directories(gtest PRIVATE third_party/googletest)
+# add_library(gtest third_party/googletest/src/gtest-all.cc)
+# target_include_directories(gtest PRIVATE third_party/googletest)
 
 include_directories(third_party/googletest/include)
 
diff --git a/crypto/CMakeLists.txt b/crypto/CMakeLists.txt
index fbfc4b27..662e4c06 100644
--- a/crypto/CMakeLists.txt
+++ b/crypto/CMakeLists.txt
@@ -110,7 +110,7 @@ add_subdirectory(x509v3)
 add_subdirectory(pkcs8)
 
 # Test support code
-add_subdirectory(test)
+# add_subdirectory(test)
 
 add_library(
   crypto
@@ -207,20 +207,20 @@ add_executable(
 target_link_libraries(refcount_test crypto)
 add_dependencies(all_tests refcount_test)
 
-# TODO(davidben): Convert the remaining tests to GTest.
-add_executable(
-  crypto_test
-
-  chacha/chacha_test.cc
-  dh/dh_test.cc
-  dsa/dsa_test.cc
-  ec/ec_test.cc
-  err/err_test.cc
-  rsa/rsa_test.cc
-
-  $<TARGET_OBJECTS:gtest_main>
-  $<TARGET_OBJECTS:test_support>
-)
-
-target_link_libraries(crypto_test crypto gtest)
-add_dependencies(all_tests crypto_test)
+# # TODO(davidben): Convert the remaining tests to GTest.
+# add_executable(
+#   crypto_test
+#
+#   chacha/chacha_test.cc
+#   dh/dh_test.cc
+#   dsa/dsa_test.cc
+#   ec/ec_test.cc
+#   err/err_test.cc
+#   rsa/rsa_test.cc
+#
+#   $<TARGET_OBJECTS:gtest_main>
+#   $<TARGET_OBJECTS:test_support>
+# )
+#
+# target_link_libraries(crypto_test crypto gtest)
+# add_dependencies(all_tests crypto_test)
