diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 2943aa8..ac30a2b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -5,7 +5,7 @@ project("Clipper")
 add_subdirectory(libs)
 add_subdirectory(libclipper)
 add_subdirectory(rest_frontend)
-# add_subdirectory(rpc_frontend)
+add_subdirectory(rpc_frontend)
 add_subdirectory(management)
 add_subdirectory(benchmarks)
 add_subdirectory(container)
diff --git a/src/libs/CMakeLists.txt b/src/libs/CMakeLists.txt
index 38476e6..d057700 100644
--- a/src/libs/CMakeLists.txt
+++ b/src/libs/CMakeLists.txt
@@ -62,4 +62,4 @@ add_subdirectory(redox)
 
 add_subdirectory(rapidjson)
 
-# add_subdirectory(grpc)
+add_subdirectory(grpc)
diff --git a/src/libs/grpc b/src/libs/grpc
--- a/src/libs/grpc
+++ b/src/libs/grpc
@@ -1 +1 @@
-Subproject commit 02f3b8604e4fccc382231259c12f4093f7d73468
+Subproject commit 02f3b8604e4fccc382231259c12f4093f7d73468-dirty
